configure

```diff
@@ -1365,7 +1365,25 @@ if (exists $opts{'gtest'} || $tests) {
     my $sys_dir = File::Spec->catdir((system_default_install_dir(),
                               $is_windows ? 'googletest-distribution' : ''));
 
+    my $use_sm_dir = 0;
     if (-f File::Spec->catfile($sm_src, $gtest_sanity)) {
+      $use_sm_dir = 1;
+    }
+    elsif (-f File::Spec->catfile($sys_dir, $gtest_sanity)) {
+      $opts{'gtest'} = $sys_dir;
+      $gtest_root = $sys_dir;
+      setEnv('GTEST_ROOT', $gtest_root);
```

Comment:
Does this setEnv() need a print statement as well (similar to below?)

Reply:
It looks like it didn't because it's a special case for safety profile.